runner-setup/Dockerfile

```diff
@@ -0,0 +1,63 @@
+# Dockerfile to setup a new runner for CI service.
+# This file has all dependencies for running the following service(s):
+#   - gen_optimized.py
+
+# Use a lightweight base image
+FROM ubuntu:24.04
+
+# Set environment variables
+# Should probably use docker compose for this
+ENV RUNNER_USER=runner
```

Comment:
Can we use `ARG` instead? See https://docs.docker.com/reference/dockerfile/#arg.

Reply:
Yes, ill update  as we will need to restart anyways due to missing perf.